app/assets/stylesheets/base/_root.scss

```diff
@@ -0,0 +1,10 @@
+:root {
+  // Colors
+  --color-original-black: #000;
+  --color-white: #fff;
+  --color-black: #15151a;
+  --color-white-180: rgba(255, 255, 255, 0.18);
+  --color-white-200: rgba(255, 255, 255, 0.2);
+  --color-white-600: rgba(255, 255, 255, 0.6);
+  --color-input-webkit-box-shadow: rgb(93, 91, 89);
```

Comment:
Prefer defining these as variables too:

```scss

$color-black: (
  default: #15151a, // or 500: #15151a to be consistent
)

$color-white: (
  180: rgba(255, 255, 255, 0.18),
  200: rgba(255, 255, 255, 0.2)
  600: rgba(255, 255, 255, 0.6)
)
```

Then it can be used here:

```scss
--color-black: map-get($color-black, 500);
```

The rationale is that the same variable (in variables) can be used outside of `:root` and it would allow a single source of truth for all variables.

Also how about defining all colors in the same unit (rgba) to avoid mix of rgb and hex colors 👍